src/detectors/ScanningStrategyDetector.ts

```diff
@@ -82,6 +85,9 @@ export class ScanningStrategyDetector implements IDetector<string, ScanningStrat
       }
```

Comment:
Line 82 after changes: It should throw an error, if the API returns 404 and we HAVE auth, so it's incorrect Auth or non-existing.